### PR TITLE
NEPT-1132: Update and fix the logic in the main form alteration process.

### PR DESCRIPTION
**Pull request by drupol**

### CODE REVIEW

File: atomium/templates/form_element_label/form_element_label.component.inc [hunk 1]
```diff
@@ -54,6 +54,7 @@ function atomium_preprocess_form_element_label(&$variables, $hook) {
     );
   }
 
+  // TODO: Should we remove this ?
```
Comment (gillesdeudon) [inline atomium/templates/form_element_label/form_element_label.component.inc]: I would say that the answer is yes as the Drupal implements already a system of hooking with its CSS classes.
Ex.: 

- `field-label`: field label;
- `node-teaser`: container of 'teaser' display of a node;
- `file-size`: container of size information displayed with the field upload widget.
- ...

File: atomium/templates/form_element/form_element.component.inc [hunk 2]
```diff
@@ -65,7 +65,6 @@ function atomium_preprocess_form_element(&$variables, $hook) {
 
   $variables['description'] = '';
   if (!empty($element['#description'])) {
-
     $variables['description'] = array(
       '#theme' => array('html_tag__' . $variables['theme_hook_original'] . '__description'),
       '#tag' => 'p',
```
Comment (gillesdeudon) [inline atomium/templates/form_element/form_element.component.inc]: This story around the use of a `p` for the description. I see in the default theme [function](https://api.drupal.org/api/drupal/includes%21form.inc/function/theme_form_element/7.x) that they set a `div` around the description.
I know that in our context that is what we need but in the context of a contrib theme, the container should be a `div`.

**Review by gillesdeudon — COMMENTED** [review]
Just 2 improvements, nothing that is blocking.

File: atomium/templates/form_element_label/form_element_label.component.inc (hunk 1 above)
Comment (drupol) [reply atomium/templates/form_element_label/form_element_label.component.inc]: Ok done.

File: atomium/templates/form_element/form_element.component.inc (hunk 2 above)
Comment (drupol) [reply atomium/templates/form_element/form_element.component.inc]: Fixed too.

**Comment by drupol** [comment]
@gillesdeudon I've updated some files, could you please give them a quick review?

Thanks.

**Comment by gillesdeudon** [comment]
Great job! I put my thumb up... In the next comment below ;-)

**Comment by gillesdeudon** [comment]
👍

**Comment by drupol** [comment]
@gillesdeudon You have to approve the changes ;)

**Review by gillesdeudon — APPROVED** [review]
Everything is ok.

**Pull request merged**